src/main/java/site/coach_coach/coach_coach_server/routine/converter/DayOfWeekSetConverter.java

```diff
@@ -0,0 +1,27 @@
+package site.coach_coach.coach_coach_server.routine.converter;
+
+import java.time.DayOfWeek;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
+
+@Converter
+public class DayOfWeekSetConverter implements AttributeConverter<Set<DayOfWeek>, String> {
+
+	@Override
+	public String convertToDatabaseColumn(Set<DayOfWeek> attribute) {
+		return String.join(",", attribute.stream().map(DayOfWeek::name).toArray(String[]::new));
+	}
+
+	@Override
+	public Set<DayOfWeek> convertToEntityAttribute(String dbData) {
+		return new HashSet<>(Arrays.asList(dbData.split(",")))
+			.stream()
+			.map(DayOfWeek::valueOf)
+			.collect(Collectors.toSet());
+	}
```

Comment:
현재 `HashSet`을 사용하여 `Set<DayOfWeek>`를 생성하고 있는데, `Arrays.asList(dbData.split(","))`로 먼저 리스트를 만들고, 이를 다시 스트림으로 처리한 후 `Set`으로 변환하고 있는 것 같네요! HashSet을 사용한 이유가 있는지 궁금합니다.

`HashSet`은 중복을 허용하지 않는 특성을 갖고 있지만, 사실 이미 `Collectors.toSet()`을 사용하면 중복이 자동으로 제거되므로, `HashSet`을 명시적으로 사용할 필요는 없어보여서요! 

또 중간에 리스트를 변환하는 부분이 불필요해보입니다... 음
```suggestion
		return Arrays.stream(dbData.split(","))
             .map(DayOfWeek::valueOf)
             .collect(Collectors.toSet());
	}
```
혹시 중간에 리스트로 변환하셨던 이유를 알 수 있을까용??

Reply:
일단은 해당 메소드 자체가 Set<DayOfWeek> 타입을 반환합니다. 그래서 return값도 동일한 Set<> 타입으로 반환하기위해 HashSet<>, TreeSet<>, LinkedHashSet<> 중에서 가장 기본적인 HashSet<>타입을 선택했습니다. 

그리고 HashSet<>의 매개변수에는 Collection 값이 필요하기에 dbDate.spli(",")를 List로 변환하였습니다.

Reply:
`Collectors.toSet()`을 사용하고 계신데, 이것 자체가 Set으로 만들어주지 않나요? 따라서 HashSet을 명시적으로 지정하지 않아도 Set타입으로 반환 가능한 것으로 알고 있습니다!

Reply:
아하, 아예 Array 대신 명시적으로 HashSet을 사용했다는 뜻이었습니다!
그런데 더 찾아보니 Collectors.toSet 메서드가 Array타입을 Set으로 변환해주는걸 알게됬네요!
한비님 말씀대로 중복요소가 있으니 수정해볼게요!